.travis.yml

```diff
@@ -27,21 +37,39 @@ env:
     - LISP=ecl OPENSSL=openssl-1.0.2q BITS=64
     - LISP=ecl OPENSSL=openssl-1.1.0j BITS=64
     - LISP=ecl OPENSSL=openssl-1.1.1a BITS=64
+    - LISP=ecl OPENSSL=libressl-2.2.7 BITS=64
+    - LISP=ecl OPENSSL=libressl-2.5.5 BITS=64
+    - LISP=ecl OPENSSL=libressl-2.6.5 BITS=64
+    - LISP=ecl OPENSSL=libressl-2.8.3 BITS=64
+    - LISP=ecl OPENSSL=libressl-3.0.1 BITS=64
+    - LISP=ecl OPENSSL=libressl-2.2.7 BITS=64
     - LISP=abcl OPENSSL=openssl-0.9.8zh BITS=64
     - LISP=abcl OPENSSL=openssl-1.0.0s BITS=64
     - LISP=abcl OPENSSL=openssl-1.0.2q BITS=64
     - LISP=abcl OPENSSL=openssl-1.1.0j BITS=64
     - LISP=abcl OPENSSL=openssl-1.1.1a BITS=64
+    - LISP=abcl OPENSSL=libressl-2.5.5 BITS=64
```

Comment:
Why no 2.2.7 for abcl, cmucl and allegro?

Reply:
Copy and paste fail. Fixed.